ndex2/client.py

```diff
@@ -414,8 +467,46 @@ def update_cx_network(self, cx_stream, network_id):
 
     def get_network_as_cx_stream(self, network_id):
         """
+
         Get the existing network with UUID network_id from the NDEx connection as a CX stream.
 
+	.. code-block:: python
+
+
+		import json
```

Comment:
I dont think `import json` is needed in this code fragment. Please check and remove if not needed.

Reply:
Could you try this code fragment without the `import json`? I'm guessing it will work